### ISSUE TITLE
Android build failed with an exception

### ISSUE THREAD

**Issue by unicorndk** [PoqXert/godot-appodeal-android-plugin#11]
Hi. I installed this plugin, but when i try to build .apk, build failed with an exception
![alt text](https://i.ibb.co/DVKfX5m/2022-07-07-151318808.png)

**Comment by Crowdel47** [comment]
Hi, I have the same error. I change "compileSdk : 30" to "31" in "config.grable" and the build is successful. But on the phone, the game does not start, tries to open and immediately closes without notification (Android 10). Any help.

**Comment by mpo4apsky** [comment]
> The game does not start, tries to open and immediately closes without notification (Android 10).

I have the same problem, did you find a way to fix this?

**Comment by Crowdel47** [comment]
No, it's something with the plugin, I disabled it and enabled admob =(

**Comment by mpo4apsky** [comment]
> No, it's something with the plugin, I disabled it and enabled admob =(

It's a pity, I hope the plugin will be fixed.

**Comment by 0hJah** [comment]
I have the same issue :(
When I compile the game with Godot 3.5 it just crashes after launch, and Godot 3.3+ just give me the same error

**Comment by mpo4apsky** [comment]
> I have the same issue :( When I compile the game with Godot 3.5 it just crashes after launch, and Godot 3.3+ just give me the same error

I did compilation on Godot 3.3+ and 3.4+, same thing.
I tried to do "adb" via usb, but I didn’t get anything sensible in the console.
Godot, after compiling, writes that you need to move the "godotappodeal" folder to the "android/plugins", although it is already there.
But it seems to me that this is just because of this very error that brought us here "compileSdk: 30"

There are also a number of compile-time permission warnings, although I have flagged them as noted in the manual for this plugin as well as the Appodeal SDK manual.

**Comment by stromperton** [comment]
the same....... someone understand what happen?
